Database/Corrections/sodQuestFixes.lua

```diff
@@ -140,6 +140,11 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410111,
```

Comment:
I just noticed that this is totally not required, since this is a real quest and not one of the "fake rune quests" we are also adding to show runes in SoD 😅 The API knows about this questId and will return `true` when using `C_QuestLog.IsQuestFlaggedCompleted(78114)` and therefore hides the quest if a player already completed it.

So feel free to remove this field for quest `78114` and `78121`.

Reply:
https://github.com/Questie/Questie/pull/5332/commits/6e704ae1a13649bfa253ed3bfb3358d0fe7744ad